File: 0412-fizz-buzz/0412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function (n) {
    let answer = [];
    for (let i = 1; i <= n; i++) {
        let res = "";
        if (i % 3 === 0) res += "Fizz";
        if (i % 5 === 0) res += "Buzz";
        answer.push(res || String(i));
    }
    return answer;
};

File: 0412-fizz-buzz/0412-fizz-buzz.cpp
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> answer;
        for (int i = 1; i <= n; i++) {
            string res;
            if (i % 3 == 0) res += "Fizz";
            if (i % 5 == 0) res += "Buzz";
            answer.push_back(res.empty() ? to_string(i) : res);
        }
        return answer;
    }
};